front/app/(a-propos)/aide-aux-elus/page.tsx

```diff
@@ -0,0 +1,131 @@
+import type { Metadata } from 'next';
+import Link from 'next/link';
+
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from '@/components/ui/accordion';
+
+export const metadata: Metadata = {
+  title: 'Aide aux élus',
+  description:
+    'Ma collectivité a une mauvaise note de transparence, pas de panique ! Éclaireur Public vous aide à améliorer la transparence des collectivités',
+};
+export default function page() {
```

Comment:
Les composants en majuscules
```suggestion
};

export default function Page() {
```

Reply:
done